src/constants/index.ts

```diff
@@ -206,3 +208,122 @@ export const TABLE_DEFAULT_ROWS_PER_PAGE = 100;
+  { label: 'East US 2 EUAP', name: 'eastus2euap' },
+  { label: 'Qatar Central', name: 'qatarcentral' },
+  { label: 'Central US (Stage)', name: 'centralusstage' },
+  { label: 'East US (Stage)', name: 'eastusstage' },
+  { label: 'East US 2 (Stage)', name: 'eastus2stage' },
+  { label: 'North Central US (Stage)', name: 'northcentralusstage' },
+  { label: 'South Central US (Stage)', name: 'southcentralusstage' },
+  { label: 'West US (Stage)', name: 'westusstage' },
+  { label: 'West US 2 (Stage)', name: 'westus2stage' },
+  { label: 'Asia', name: 'asia' },
+  { label: 'Asia Pacific', name: 'asiapacific' },
+  { label: 'Australia', name: 'australia' },
+  { label: 'Brazil', name: 'brazil' },
+  { label: 'Canada', name: 'canada' },
+  { label: 'Europe', name: 'europe' },
+  { label: 'France', name: 'france' },
+  { label: 'Germany', name: 'germany' },
+  { label: 'Global', name: 'global' },
+  { label: 'India', name: 'india' },
+  { label: 'Japan', name: 'japan' },
+  { label: 'Korea', name: 'korea' },
+  { label: 'Norway', name: 'norway' },
+  { label: 'Singapore', name: 'singapore' },
+  { label: 'South Africa', name: 'southafrica' },
+  { label: 'Switzerland', name: 'switzerland' },
+  { label: 'United Arab Emirates', name: 'uae' },
+  { label: 'United Kingdom', name: 'uk' },
+  { label: 'United States', name: 'unitedstates' },
+  { label: 'United States EUAP', name: 'unitedstateseuap' },
+  { label: 'East Asia (Stage)', name: 'eastasiastage' },
+  { label: 'Southeast Asia (Stage)', name: 'southeastasiastage' },
+  { label: 'East US STG', name: 'eastusstg' },
+  { label: 'South Central US STG', name: 'southcentralusstg' },
+  { label: 'North Central US', name: 'northcentralus' },
+  { label: 'West US', name: 'westus' },
+  { label: 'Jio India West', name: 'jioindiawest' },
+  { label: 'Central US EUAP', name: 'centraluseuap' },
+  { label: 'West Central US', name: 'westcentralus' },
+  { label: 'South Africa West', name: 'southafricawest' },
+  { label: 'Australia Central', name: 'australiacentral' },
+  { label: 'Australia Central 2', name: 'australiacentral2' },
+  { label: 'Australia Southeast', name: 'australiasoutheast' },
+  { label: 'Japan West', name: 'japanwest' },
+  { label: 'Jio India Central', name: 'jioindiacentral' },
+  { label: 'Korea South', name: 'koreasouth' },
+  { label: 'South India', name: 'southindia' },
+  { label: 'West India', name: 'westindia' },
+  { label: 'Canada East', name: 'canadaeast' },
+  { label: 'France South', name: 'francesouth' },
+  { label: 'Germany North', name: 'germanynorth' },
+  { label: 'Norway West', name: 'norwaywest' },
+  { label: 'Switzerland West', name: 'switzerlandwest' },
+  { label: 'UK West', name: 'ukwest' },
+  { label: 'UAE Central', name: 'uaecentral' },
+  { label: 'Brazil Southeast', name: 'brazilsoutheast' },
+];
+
+const generateGCPRegions: any = () => {
+  const servers = [
+    { name: 'asia-east', count: [1, 2], letters: ['a', 'b', 'c'] },
```

Comment:
I just got these values from the GCP region list in google's documetnation for what the server options are. There are a lot of "asia-east1-a" "asia-east2-a", etc. Instead of copying them all in to a giant variable, I compacted it a little so we generate them instead.
https://cloud.google.com/compute/docs/regions-zones

Unfortunately, this list also changes in the follow up PR too! I didn't realize jade-repo is not expecting letters at all...

Reply:
Could you add that as a comment? I think that is valuable information (showing what the compound version is)

Reply:
done!